moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1377,19 +1490,62 @@ void PlanningSceneMonitor::configureDefaultPadding()
   }
 
   // Ensure no leading slash creates a bad param server address
-  const std::string robot_description =
+  static const std::string robot_description =
       (robot_description_[0] == '/') ? robot_description_.substr(1) : robot_description_;
 
-  nh_.param(robot_description + "_planning/default_robot_padding", default_robot_padd_, 0.0);
```

Comment:
These have direct corresponding functions, we don't need to check `has_parameter` everywhere.